README.md

```diff
@@ -87,7 +87,7 @@ java_war(
     data_path = "webapp",
     deps = [
         "//external:appengine/java/api",
-        "//external:javax/servlet/api",
+        "@io_bazel_rules_appengine//appengine:javax.servlet.api",
```

Comment:
oh right forgot about it.

LGTM

On Wed, Mar 23, 2016 at 5:09 PM Kristina <notifications@github.com> wrote:

> In README.md
> <https://github.com/bazelbuild/rules_appengine/pull/5#discussion_r57187718>
> :
>
> > @@ -87,7 +87,7 @@ java_war(
> >      data_path = "webapp",
> >      deps = [
> >          "//external:appengine/java/api",
> > -        "//external:javax/servlet/api",
> > +        "@io_bazel_rules_appengine//appengine:javax.servlet.api",
>
> You made it a neverlink target (ac05897
> <https://github.com/bazelbuild/rules_appengine/commit/ac058970bc8d781d04e570358a1fcc073908f0d3>),
> which seemed worth preserving.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bazelbuild/rules_appengine/pull/5/files/fd5df80a72cdcab579302456078dceb4a797c610#r57187718>
>